internal/compiler-interface/src/main/java/xsbti/compile/IncOptionsUtil.java

```diff
@@ -7,16 +7,17 @@
 
 package xsbti.compile;
 
+import xsbti.Maybe;
+
 import java.io.File;
 import java.util.HashMap;
 import java.util.Map;
-import xsbti.Maybe;
 
 /**
- * Helper object for xsbti.compile.IncOptions
+ * Define a helper class to instantiate {@link IncOptions}.
```

Comment:
In what way does IncOptionsUtil "define a helper class"?

It _is_ a helper class/object.